h2o-docs/src/product/explainability.rst

```diff
@@ -0,0 +1,213 @@
+Model Explainability
+====================
+
+H2O offers a convenient interface to many explainabilty methods in H2O.  The functions work for individual models, as well a list of models or an `H2O AutoML object <automl.html>`__
+
+
+Explainability Interface
+------------------------
+
+The H2O explainability interface is designed to be automatic -- all of the explainability feedback is generated with a single function, ``h2o.explain()``, which takes an object (one of the following: an H2O model, a list of models, an H2OAutoML object, an H2OAutoML Leaderboard slice), and a test frame.
+
+
+.. tabs::
+   .. code-tab:: r R
+
+        # Explain a model
+        exm <- h2o.explain(model, test)
+        print(exm)
+
+        # Explain an AutoML object
+        exa <- h2o.explain(aml, test)
+        print(exa)
+
+
+   .. code-tab:: python
+
+        # Explain a model
+        exm = model.explain(test)
+        print(exm)
+
+        # Explain an AutoML object
+        exa = aml.explain(test)
+        print(exa)
```

Comment:
Remove `print` in python.

Reasons mentioned above.

Reply:
done